### DISCUSSION TITLE
The first configuration, the Black Diamond

### DISCUSSION THREAD

**Discussion by mickleby** [romkatv/powerlevel10k#2609] [General]
I'm curious to understand why you link https://graphemica.com/%E2%97%86, i.e. U+25C6, and yet display \uE0B2\uE0B0.

**Comment by romkatv** [comment]
What would you suggest as an alternative?

**Reply by mickleby** [reply]
It seems like you are checking for Private Use Area glyphs. I don't understand why, but I assume you will defer to them if they exist.

**Reply by romkatv** [reply]
I'm not sure which alternative you are suggesting. What can I link to instead of https://graphemica.com/%E2%97%86?

**Reply by romkatv** [reply]
And yes, I need to find out whether the two specific private use area glyphs render as two specific triangle shapes.

**Reply by mickleby** [reply]
Ah, no my question is about the U+E0B2 and U+E0B0. Perhaps YOU will use them elsewhere in the code? I'll look for that now.

**Reply by romkatv** [reply]
You can reply Yes to the first question and see them used right away.

**Reply by mickleby** [reply]
So you're simply checking that they are not already assigned?

**Reply by romkatv** [reply]
No. https://github.com/romkatv/powerlevel10k/discussions/2609#discussioncomment-8819712

**Reply by romkatv** [reply]
If you simply use the wizard, I'm sure you'll see the logic behind it. Try answering Yes.

**Comment by mickleby** [comment]
I installed and switched to Nerd Fonts before running the configure wizard. Now I understand much better!🤣

I *really* appreciate your time and patience. Cheers!
